### PR TITLE
add vcan support

### PR DESCRIPTION
**Pull request by thomas-roos**

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -31,9 +31,18 @@ trap cleanup EXIT SIGINT SIGTERM
 
 # Main execution
 main() {
+    # make vcan0 interface
```
Comment (nenadilic84) [inline start.sh:34]: I would actually remove this from here

Comment (nenadilic84) [reply start.sh:34]: We have the instructions to set this script up

Comment (thomas-roos) [reply start.sh]: removed this, as discussed.

File: start.sh [hunk 2]
```diff
@@ -43,7 +47,7 @@ main() {
     source ~/ros2_ws/install/setup.bash
 
     cd carla-client
-    pids+=($(start_process "python3 ./manual_control_steeringwheel.py --sync --rolename ego_vehicle --filter vehicle.tesla.model3 -i can0" "../$log_dir/control.log"))
+    pids+=($(start_process "python3 ./manual_control_steeringwheel.py --sync --rolename ego_vehicle --filter vehicle.tesla.model3 -i vcan0" "../$log_dir/control.log"))
```
Comment (nenadilic84) [inline start.sh:50]: let's leave this can0 for now. I'll create another PR to fix this

Comment (thomas-roos) [reply start.sh:50]: ok

**Pull request merged**